### PR TITLE
Devops/251 global config

### PR DESCRIPTION
**Pull request by andrea-williams**
Added global.config.yaml to Openshift templates & updated README to describe config keys

### CODE REVIEW

File: openshift/templates/global.config.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- kind: ConfigMap
+  apiVersion: v1
+  metadata:
+    name: wps-api-global
+    labels:
+      app.kubernetes.io/part-of: wps-api
+  data:
+    env.origins: wps-web-*.pathfinder.gov.bc.ca
+    env.spotwx-url: 'https://spotwx.com/services/api.php'
+    env.wfwx-auth-url:
```
Comment (Sybrand) [inline openshift/templates/global.config.yaml:15]:
```suggestion
    env.wfwx-auth-url: ${WFWX-AUTH-URL}
```

File: openshift/templates/global.config.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    env.wfwx-base-url:
+    env.wfwx-user:
+- kind: Secret
+  apiVersion: v1
+  metadata:
+    name: wps-api-global
+  type: Opaque
+  data:
+    spotwx-api-key:
+    wfwx-secret:
```
Comment (Sybrand) [inline openshift/templates/global.config.yaml:25]:
```suggestion
    wfwx-secret:
  parameters:
  - description: Wildfire One API authentication URL
  - name: WFWX-AUTH-URL
```

File: README.md [hunk 3]
```diff
@@ -131,6 +131,23 @@ Or enforce by running [scripts/test.sh](scripts/test.sh) as part of your ci/cd p
 
 ![FWI calculator container diagram](container_diagram.png)
 
+## Config
```
Comment (Sybrand) [inline README.md:134]: Let's rather just be specific in mentioning the existence of the global.config.yaml and ????secret???.yaml file, and have the descriptions in the yaml files themseles. We could add here an example of how to apply the template, e.g:

```
oc -n openshift-project process -f openshift/templates/global.config.yaml -p WFWX-AUTH-URL=https://spotwx.com/
```

File: openshift/templates/global.config.yaml (hunk 2 above)
Comment (Kyubinhan) [reply openshift/templates/global.config.yaml:25]: Can you write `parameters` field right before `objects` field for consistency?

**Pull request merged**